brewtils/rest/client.py

```diff
@@ -366,6 +366,7 @@ def get_systems(self, **kwargs):
         """Perform a GET on the System collection URL
 
         Args:
+            filter_latest (bool): Filter latest system versions
```

Comment:
Maybe I'm missing something. How is filter_latest being provided to the system_url?

Reply:
If it's in the kwargs why put it here?

Reply:
Just for documentation. I added it to be explicit that it is a new field that we support.